views/partials/post/local-events-cards.blade.php

```diff
@@ -15,7 +15,7 @@
                    @slot('subHeading')
                    @typography(['variant' => 'meta', 'element' => 'p'])
                         @icon(['icon' => 'date_range']) @endicon
-                        @date(['action' => 'formatDate', 'format' => 'D d M Y H:i' ,'timestamp' => $post->startDate])@enddate
+                            {{$post->startDate}}
```

Comment:
Use date component, but leave 'action' blank. This will print time html tag but not manipulate the date string.